db_client/alembic/versions/0041_add__document_to_unfccc_tax.py

```diff
@@ -0,0 +1,86 @@
+"""Add _document and _document role to UNFCCC taxonomy.
+
+Revision ID: 0041
+Revises: 0040
+Create Date: 2024-07-02 14:24:00.140151
+
+"""
+
+from alembic import op
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import Session
+
+from db_client.data_migrations.taxonomy_utils import read_taxonomy_values
+from db_client.utils import get_library_path
+
+# revision identifiers, used by Alembic.
+revision = "0041"
+down_revision = "0040"
+branch_labels = None
+depends_on = None
+
+Base = automap_base()
+
+INTL_AGREEMENTS = "Intl. agreements"
+
+TAXONOMY_DATA = [
+    {
+        "key": "author_type",
+        "allow_blanks": False,
+        "allowed_values": ["Party", "Non-Party"],
+    },
+    {
+        "key": "author",
+        "allow_blanks": False,
+        "allow_any": True,
+        "allowed_values": [],
+    },
+    {
+        "key": "event_type",
+        "filename": f"{get_library_path()}/alembic/versions/data/0041/event_type_data.json",
+        "file_key_path": "name",
+        "allow_blanks": True,
+    },
+    {
+        "key": "_document",
+        "taxonomy": [
+            {
+                "key": "role",
+                "filename": f"{get_library_path()}/alembic/versions/data/0041/document_role_data.json",
```

Comment:
Not sure if all these document roles are applicable, neither the event_types - let's ask @alaniswright

Reply:
You could also have a rummage in the prod db

Reply:
Have asked Alan - waiting for a response from policy. Peter and I have agreed to progress and handle any change in these separately.